proxy/envoy/testdata/rds-ingress-ssl.json.golden

```diff
@@ -11,7 +11,7 @@
       "cluster": "out.81c187d71467b1608736c57bb0734f9ef9b68f7d",
       "opaque_config": {
        "mixer_control": "on",
-       "mixer_forward": "off"
+       "mixer_forward": "on"
```

Comment:
For ingress, we actually want both: first we check against the cluster ingress incoming request, then we forward the source attributes set to the ingress service to the backends.
Is there a problem with doing both?